nvflare/private/fed/server/job_runner.py

```diff
@@ -123,6 +125,8 @@ def _start_run(self, run_number, job: Job, client_sites: list, fl_ctx: FLContext
         if not replies:
             raise RuntimeError("Failed to start the App to the clients")
 
+        display_sites = ",".join(list(client_sites.keys()))
```

Comment:
is this arg `client_sites` a list or a dict?
The type hinting is saying it is a list

Reply:
Fixed the type hint.